libseccomp/src/error.rs

```diff
@@ -40,6 +40,8 @@ pub enum SeccompErrno {
     ERANGE,
     /// Unable to load the filter due to thread issues.
     ESRCH,
+    /// Libseccomp attempted to access a resource that is in use by another process
+    EBUSY,
```

Comment:
I don't think that libseccomp ever returns this with that meaning. See

https://github.com/seccomp/libseccomp/blob/f1c3196d9b95de22dde8f23c5befcbeabef5711c/src/api.c#L52-L118

Instead it only happens with sys_raw_rc where `seccomp(2)` says

> **EBUSY** 
> While installing a new filter, the  SECCOMP_FILTER_FLAG_NEW_LISTENER flag was specified, but a previous filter had already been installed with that flag.

Reply:
Mybad, i even updated the actual message, missed the doc comment. I'll also throw up a changelog update